security/cyrus-sasl2/Portfile

```diff
@@ -30,26 +29,24 @@ master_sites            ${homepage}/releases/ \
                         ftp://ftp.cyrusimap.org/cyrus-sasl/
 distname                cyrus-sasl-${version}
 
-checksums               rmd160  861a06d663cf3da37a198d0f971d99b249b5f4b8 \
-                        sha256  8fbc5136512b59bb793657f36fadda6359cae3b08f01fd16b3d406f1345b7bc3 \
-                        size    5220231
+checksums               rmd160  7f5e4bd2ba1974f0b96ba5a659165f4484d550ce \
+                        sha256  26866b1549b00ffd020f188a43c258017fa1c382b3ddadd8201536f72efb05d5 \
+                        size    4111249
 
 depends_lib             path:lib/libssl.dylib:openssl \
                         port:zlib
 
 default_variants        +kerberos
 
-patchfiles              patch-libsasl2.pc.in.diff \
-                        patch-ltconfig_rpath_fallback.diff \
-                        patch-utils_Makefile.in.diff \
-                        patch-sasl.h-size_t.diff
-
 configure.args          --mandir=${prefix}/share/man \
                         --with-openssl=${prefix} \
                         --with-dbpath=${prefix}/etc/sasldb2 \
                         --with-plugindir=${prefix}/lib/sasl2 \
                         --with-saslauthd=${prefix}/var/state/saslauthd \
                         --with-pwcheck=${prefix}/var/pwcheck \
+                        --disable-obsolete_cram_attr \
+                        --disable-obsolete_digest_attr \
+                        --enable-static \
```

Comment:
`--enable-static` was removed a long time ago. See https://trac.macports.org/ticket/22093. Does it work now? Your commit message does not mention that you are turning this back on.